org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/preferences/CodeGenerationTemplate.java

```diff
@@ -161,9 +162,10 @@ public Template createTemplate(IJavaProject project) {
  * Preference key names, internal for now.
  */
 class CodeTemplatePreferences {
-	private static final String CODETEMPLATES_PREFIX = "java.codetemplates."; //$NON-NLS-1$
-	public static final String COMMENT_SUFFIX = ".comment"; //$NON-NLS-1$
-	public static final String BODY_SUFFIX = ".body"; //$NON-NLS-1$
+	private static final String CODETEMPLATES_PREFIX = "org.eclipse.jdt.ui.text.codetemplates."; //$NON-NLS-1$
```

Comment:
why do we need a jdt.ui namespaced prefix?

Reply:
This is in order to access the following code template definitions, https://github.com/eclipse/eclipse.jdt.ui/blob/master/org.eclipse.jdt.ui/templates/default-codetemplates.xml . Currently they're defined in jdt.ui. Perhaps they could be moved in the future and even have their namespace changed but haven't looked into it yet.

I'll definitely take care of the other issues mentioned.